src/bundle/ui-dev/src/modules/multi-file-upload/multi.file.upload.module.js

```diff
@@ -110,6 +110,7 @@ export default class MultiFileUploadModule extends Component {
 
     handleDropOnWindow(event) {
         this.preventDefaultAction(event);
+        event.stopImmediatePropagation();
```

Comment:
Note: MFU is rendered twice in Sub-items, because _extraAction_ items are rendered twice - once in the normal list and once in the popup list. Even though at the given time one of them always is hidden, they are always both rendered at the same time.
Because of this we want to prevent one of them from handling file d&d, so that it is handled only by one of them. Regular `event.stopPropagation()` does not work, because both handlers are attached to the same node thus we need in this case `event.stopImmediatePropagation()`.
ref. https://developer.mozilla.org/en-US/docs/Web/API/Event/stopImmediatePropagation